src/pages/library/dataBrowser-utils.js

```diff
@@ -100,6 +103,11 @@ const extractTags = dataset => {
   }
 }
 
+export const getDatasetsPath = ({ user: { id } }) => {
+  return isDataBrowserVisible() && catalogPreviewStore.get()[id] ?
```

Comment:
Fixed this bug. I'd be okay with making that change to use a new configOverridesStore, just trying to think through what the pros and cons of this would be. I'm using the localstorage because I think that's also tied to user... Do we care if someone logs out, the next person who uses the same browser/computer ends up getting the same state? Since you can toggle it back easily, I dont think it's that important 🤔 

The other thing that I'm also thinking about is that this is only supposed to be around as long as we are maintaining both versions. The catalog is supposed to eventually replace it, which means this localstorage vs configoverrides is eventually all going to have to go away.

Dunno! Looking for other opinions on this too! Thanks :)